t/app/controller/report_updates.t

```diff
@@ -387,7 +387,7 @@ for my $test (
         is $add_alerts, $details->{add_alert} ? 1 : 0, 'do not sign up for alerts';
 
         $mech->get_ok( $url . $url_token );
-        $mech->content_contains("/report/$report_id#update_$update_id");
+        $mech->content_contains("Thank you for updating this issue");
```

Comment:
This test (and the one below) was correct, in that this URL should still appear in the resultant page. The fact it doesn't is a bug in the template, not the test, I'll note the place in the template.